gateway-ha/src/test/java/io/trino/gateway/TrinoGatewayRunner.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.gateway;
+
+import io.trino.gateway.ha.HaGatewayLauncher;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.TrinoContainer;
+
+import java.util.List;
+
+import static org.testcontainers.utility.MountableFile.forClasspathResource;
+
+public final class TrinoGatewayRunner
+{
+    private TrinoGatewayRunner() {}
+
+    public static void main(String[] args)
+            throws Exception
+    {
+        TrinoContainer trino1 = new TrinoContainer("trinodb/trino");
```

Comment:
Should we fix the version or provide an option to set it .. maybe even different ones for the two containers?

Reply:
Fixing version sounds more stable. 
Then we could write sth like `compatible with Trino 435+`
- I rarely think think there would be any compatibility issue, but some drastic API change might happen

What would be reasons for having different versions? A testbed to check if it is allright to upgrade version?

Reply:
Yeah .. we have to add this to the docs yet as well .. on my plate..

Reply:
This query runner is just for local testing and isn't ensured in CI. We shouldn't document the version.